Makefile

```diff
@@ -44,6 +44,7 @@ build-app:
 	make -C SampleApps/$(APP_NAME) build-shallow
 
 archive:
+	bundle exec pod update
```

Comment:
Can we add the make target `pod-update` as dependency to archive?

Reply:
`pod-update` doesn't use Bundler. However, I can change `pod-update` to use Bundler instead if that makes sense.